run.sh

```diff
@@ -22,6 +22,8 @@ export DEV_ENVIRONMENT="Y"
 export RELOAD_TEMPLATES="Y"
 export TEMPLATES_DIR=src/main/web/templates/handlebars
 export ENABLE_COVID19_FEATURE=false
+export ENABLE_NAVIGATION="Y"
+export TOPICS_SERVICE_URL="http://localhost:25300"
```

Comment:
I can see by default we're bypassing the router, would we do this in prod?

Reply:
See dp-configs merged march 24th need to update staging/prod when ready to deploy